### PR TITLE
chore(workflow-beta1-test): create chart with immutable git shas

### PR DESCRIPTION
**Pull request by sgoings**

### CODE REVIEW

File: workflow-beta1-test/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: workflow-dev
```
Comment (bacongobbler) [inline workflow-beta1-test/Chart.yaml:1]: should be renamed to workflow-beta1-test

Comment (sgoings) [inline workflow-beta1-test/Chart.yaml:1]: Good call

**Comment by jchauncey** [comment]
@sgoings merge it :ship:

File: workflow-beta1-test/Chart.yaml (hunk 1 above)
Comment (sgoings) [inline workflow-beta1-test/Chart.yaml:1]: I'll add a pr to fix this once I'm back at home.

**Pull request merged**